clusterloader/docs/vision.md

```diff
@@ -0,0 +1,335 @@
+# Cluster loader vision
+
+Author: wojtek-t
+
+Last update time: 9th May 2018
+
+## Background
+
+As of 31/03/2018, all our scalability tests, are regular e2e tests written in Go.
+It makes them really unfriendly for developers not working on scalability who just
+want to load tests Kubernetes features they are working on. Doing so in many
+situation requires understanding how those tests really work, modify their code
+to test the new feature and only then run and debug tests (or create load test for
```

Comment:
You could move this long sentence inside brackets to new sentence?

Reply:
done